src/devices.coffee

```diff
@@ -30,13 +30,21 @@ module.exports =
         client.post 'device/', data, (err, res, body) ->
             if err
                 callback err
+            else if body.error is 'This name is already used'
+                deviceName = "#{options.deviceName}-2"
+                deviceName = deviceName.replace /-(\d+)-2/, (_, suffix) ->
```

Comment:
Well, it adds a new function, which I wanted to avoid. But I think it's more important to be consistent with cozy-mobile. So, let's do the same.